logbook-spring-boot3-autoconfigure/src/main/resources/META-INF/spring.factories

```diff
@@ -0,0 +1 @@
+org.springframework.boot.autoconfigure.EnableAutoConfiguration = org.zalando.logbook.autoconfigure.LogbookAutoConfiguration
```

Comment:
I think this won't work in SpringBoot 3. We need `META-INF/spring/org.springframework.boot.autoconfigure.AutoConfiguration.imports`. See https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-3.0.0-M5-Release-Notes#auto-configuration-registration

Reply:
done